services/src/main/java/org/keycloak/services/resources/admin/IdentityProvidersResource.java

```diff
@@ -92,20 +97,19 @@ public IdentityProvidersResource(RealmModel realm, KeycloakSession session, Admi
     @NoCache
     @Produces(MediaType.APPLICATION_JSON)
     @Tag(name = KeycloakOpenAPI.Admin.Tags.IDENTITY_PROVIDERS)
-    @Operation( summary = "Get identity providers")
-    public Response getIdentityProviders(@Parameter(description = "Provider id") @PathParam("provider_id") String providerId) {
+    @Operation( summary = "Get the identity provider factory for that provider id")
+    public IdentityProviderFactory getIdentityProviders(@Parameter(description = "The provider id to get the factory") @PathParam("provider_id") String providerId) {
```

Comment:
Minor: Thanks for updating the outdated javadoc and description. But method name is still slightly outdated. This minor change to update it.

```suggestion
    public IdentityProviderFactory getIdentityProviderFactory(@Parameter(description = "The provider id to get the factory") @PathParam("provider_id") String providerId) {
```

Reply:
Done!